SNsimulation/R/social_network_simulation.R

```diff
@@ -0,0 +1,36 @@
+
+# User input --------------------------------------------------------------
+
+
+
+# Make graph --------------------------------------------------------------
+
+#' @export
+create_graph <- function (n = 10){# n is number of friends
```

Comment:
Nice that you have a lot of commenting! I am not sure how the user could specify the specific edges, so which of the people in his/her social network are in contact with which other people specifically.